### ISSUE TITLE
fatal: detected dubious ownership in repository at '/github/workspace'

### ISSUE THREAD

**Issue by aerphanas** [bridgelightcloud/github-mirror-action#5]
How to reproduce :
Steps:
1. Click [Use this template](https://github.com/bridgelightcloud/github-mirror-template/generate).
2. Name your repository
3. Edit `.github/workflows/mirror.yml` and enter the URL you wish to mirror where it says `origin: ''`
4. Commit your change

![Screenshot from 2022-12-21 18-45-22](https://user-images.githubusercontent.com/61340494/208898113-38b6d257-6fc3-46e2-9d5f-c30625d984c5.png)

**Comment by cpauvert** [comment]
Hi, 
I had the same issue, it was fixed in the pull request #4 by @kaikokert, but since it was not incorporated yet, I edited the line in my `.github/workflows/mirror.yml` to:

```yaml
        uses: kaidokert/github-mirror-action@v4
```
instead of 

```yaml
        uses: bridgelightcloud/github-mirror-action@v1
```

@amadigan thanks for developing this action, is there any plan to merge the PR #4  to fix this issue?

Best,

**Comment by amadigan** [comment]
Sorry, I’ve been taking a long break, but the change looks good and has been merged.
